tests/test_monitor.py

```diff
@@ -0,0 +1,116 @@
+"""Unit tests for monitor module."""
+
+import unittest
+from unittest import mock
+from typing import Optional, Tuple
+
+from PyQt5.QtWidgets import QApplication
+
+from iquip import monitor
+
+class TestMonitor(unittest.TestCase):
+    """Unit tests for Monitor class."""
+
+    def test_init_initial_value(self):
+        for value in (True, None, 1.0, "value", object()):
+            mon = monitor.Monitor(initial_value=value)
+            self.assertEqual(mon._value, value)
+
+    def test_init_updated_callback(self):
```

Comment:
How about checking the default value of `update_callback`?
If you think it isn't necessary, never mind!

Reply:
Okay, I will add that test!